src/Shared/Components/InfoIconTippy/InfoIconTippy.tsx

```diff
@@ -0,0 +1,53 @@
+import { TippyCustomized } from '../../../Common/TippyCustomized'
+import { TippyTheme, TippyCustomizedProps } from '../../../Common/Types'
+import { ReactComponent as ICHelpOutline } from '../../../Assets/Icon/ic-help-outline.svg'
+import { ReactComponent as HelpIcon } from '../../../Assets/Icon/ic-help.svg'
+
+const InfoIconTippy = ({
+    heading,
+    infoText,
+    iconClass = 'fcv-5',
+    documentationLink,
+    documentationLinkText,
+    additionalContent,
+    className = 'icon-dim-16',
+    placement = 'bottom',
+    dataTestid,
+}: Pick<
```

Comment:
Move it to interface like export interface InfoIconTippyProps extends Pick< TippyCustomizedProps , 'heading' | ... > {dataTestid?: string}